### DISCUSSION TITLE
大佬能不能给通知脚本加个注释啊

### DISCUSSION THREAD

**Discussion by zthuii** [MikeWang000000/Natter#80] [Q&A]
麻烦大佬，通知脚本实现 Cloudflare 的跳转功能的那个实在看不懂，不知道改哪里，希望大佬能指导一下，给加一个注释

**Comment by MikeWang000000** [comment]
个人需要填写的只有 `cf_` 开头的一部分变量。
其他都是根据 [Cloudflare API Documentation](https://developers.cloudflare.com/api/) 中的说明编写的。

总的流程就是这两个成员方法：
https://github.com/MikeWang000000/Natter/blob/27d7c1d18ae70a2d6027b843ed162e3f4e28a818/natter-docker/nginx-cloudflare/cf-redir.py#L38-L47
https://github.com/MikeWang000000/Natter/blob/27d7c1d18ae70a2d6027b843ed162e3f4e28a818/natter-docker/nginx-cloudflare/cf-redir.py#L49-L59

因为比较简单，所以个人感觉没有加注释的必要。如果有不明白的地方，可以具体指出位置，我来回答。
